tokyomovie/tokyomovie
# Feat/event pages

# Description

Admin pages for movies & events

# Notes

- Fixes height for layouts on admin pages
- Adds `eventStartsAt` to events table
- Adds /admin/event /admin/movie
- Adds a basic /admin navigation page

# How to check

- Go to /admin/movie, create a movie
- Go to /admin/event, create an event

Forms are starting to develop a pattern to them, so if you see something funky, or you think something can be simplified plz let me know.
